web/src/realm_user_settings_defaults.ts

```diff
@@ -43,7 +43,7 @@ export type RealmDefaultSettings = {
     wildcard_mentions_notify: boolean;
 };
 
-export let realm_user_settings_defaults = {} as RealmDefaultSettings;
+export let realm_user_settings_defaults: RealmDefaultSettings;
```

Comment:
This uncovered a bug in #25759 where `browser_history` is trying to access `user_settings.default_view` before `user_settings.initialize_user_settings` is called: https://github.com/zulip/zulip/pull/25759#discussion_r1320422274.